New design launch review (#6615)
GoogleChrome/web.dev: src/site/content/en/index.md

---
layout: 'homepage-next'
title: 'web.dev'
intro:
  eyebrow: 'About web.dev'
  title: "Let's build the future of the web, together"
  summary: |

>     'Take advantage of the latest modern technologies to build amazing web experiences for everyone'

Reviewer: ```suggestion
    'Take advantage of the latest modern technologies to build amazing web experiences for everyone.'
```

Reviewer 2: These single quotes are displayed on the home page

Author: fixed by Sam

  buttonText: 'Learn more'
  buttonUrl: '/about/'
  image: 'https://web-dev.imgix.net/image/VbAJIREinuYvovrBzzvEyZOpw5w1/B9KWrQo39fXbKSVYjz1e.svg'
  imageWidth: '330'
  imageHeight: '210'
featureCard:
  theme: 'dark'
  title: 'Explore structured learning paths to learn new skills'
  eyebrow: 'Learn'
  url: '/learn'
  background: 'https://web-dev.imgix.net/image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/yJSJQfrrK4pVh44yuMZV.svg'
promoPanel:
  title: 'Chrome Dev Summit 2021'
  summary: 'Join the Chrome team for workshops, office hours, and learning lounges taking place throughout November.'
  primaryButtonUrl: 'https://events.withgoogle.com/chrome-dev-summit-2021/registrations/new/details/'

>   primaryButtonText: 'REQUEST AN INVITE'

Reviewer: we should probably make this text all-caps using `text-transform` rather than have the string in caps. I think some screen readers announce all caps text differently.

Author: Fixed

  secondaryButtonUrl: 'https://developer.chrome.com/devsummit/schedule/'
  secondaryButtonText: 'VIEW SCHEDULE'
  image: 'https://web-dev.imgix.net/image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/KwS4L954TzN3KK3pWmtA.svg'
  imageWidth: '530'
  imageHeight: '483'
pickedCaseStudyUrl: '/mainline-mensware/'
pickedCaseStudyTheme: 'light'
pageSpeed:
  eyebrow: 'PageSpeed Insights'
  title: 'Measure page quality with PageSpeed Insights'
  summary: |
    See how well your website performs. Then, get tips to improve your user experience.
  image: 'https://web-dev.imgix.net/image/VbAJIREinuYvovrBzzvEyZOpw5w1/6Ye3EKc6quNaopPrigno.png'
  imageWidth: '546'
  imageHeight: '421'

>   primaryButtonText: 'Try Page Speed Insights'

Reviewer 2: ```suggestion
  primaryButtonText: 'Try PageSpeed Insights'
```

  primaryButtonUrl: '/measure'
  secondaryButtonText: 'Learn more'

>   secondaryButtonUrl: '#'

Reviewer 3: Can we remove this button if it doesn't go anywhere? (Same for all lang pages)

courses:

>   title: 'Featured Courses'

Reviewer 2: ```suggestion
  title: 'Featured courses'
```

  summary: 'Explore our structured learning paths to discover everything you need to know about building for the modern web.'
caseStudies:

>   title: 'Case Studies'

Reviewer 2: ```suggestion
  title: 'Case studies'
```

  summary: 'Learn why and how other developers have used the web to create amazing web experiences for their users.'
  buttonText: 'See all'
  buttonUrl: '/tags/case-study/'
podcasts:

>   title: 'Podcasts & Shows'

Reviewer: ```suggestion
  title: 'Podcasts and Shows'
```

Reviewer: https://developers.google.com/style/text-formatting?hl=en (don't use ampersands)
